### PR TITLE
Describe Amazon S3 file access in readme

### PR DESCRIPTION
**Pull request by Borewit**
As suggested in #331.

### CODE REVIEW

**Comment by Borewit** [comment]
Thanks @sindresorhus!

**Comment by sindresorhus** [comment]
Nice! I would add short sentence in the `@tokenizer/s3` readme about this use-case and link to https://github.com/sindresorhus/file-type#filetypefromtokenizertokenizer

File: readme.md [hunk 1]
```diff
@@ -215,6 +215,31 @@ const audioTrackUrl = 'https://test-audio.netlify.com/Various%20Artists%20-%2020
...
+const S3 = require('aws-sdk/clients/s3');
+const {makeTokenizer} = require('@tokenizer/s3');
+
+(async () => {
+	// Initialize the S3 client
+	const s3 = new S3();
+
+	// Initialize the S3 tokenizer.
+	const s3Tokenizer = await makeTokenizer(s3, {
+		Bucket: 'affectlab',
```
Comment (sindresorhus) [inline readme.md]: Why are these PascalCase? `Bucket` => `bucket`, if you're taking API feedback.

Comment (Borewit) [reply readme.md]: That is inherited from the Amazon S3 client API.

Comment (sindresorhus) [reply readme.md]: Ah ok

**Pull request merged**